misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -469,7 +469,7 @@ def handle_junit_error(
                 location: str = error.file
                 location_url = None
 
-            handle_log_error(error.match, location, location_url)
+            handle_error(error.match.decode("utf-8"), None, location, location_url)
```

Comment:
Should we hand through `internal_error_type="JUNIT_FAILURE"`?
(See previous line 453.)

Reply:
Was that useful? I think it's more useful to have the currently set internal error types.

Reply:
What do you mean? This is the internal error type.
It will allow to filter JUnit failures and restrict the search to those entries; I don't know yet how useful this will be.

Ok if you want to keep it as it is.